docs/develop/01_sdk/02_cookbook/01_dids/01_light_did_creation.md

```diff
@@ -8,14 +8,14 @@ import TsJsBlock from '@site/src/components/TsJsBlock';
 import LightDidSimple from '!!raw-loader!@site/code_examples/core_features/did/01_light_did_simple.ts';
 import LightDidComplete from '!!raw-loader!@site/code_examples/core_features/did/02_light_did_complete.ts';
 
-The creation of a light DID requires the generation of some keying material for keys that are to be used for authentication, encryption, attestation, or delegation.
-For the sake of ease of use, the example snippets below show how to use a Keyring, provided also by the `@polkadot/api` library, to generate key pairs that are kept in memory and disappear at the end of the program execution, unless saved to some persistent storage.
+The creation of a light DID requires the generation of some keying material for keys that are to be used for authentication and encryption.
+For the sake of ease of use, the example snippets below show how to use keys generated with a `Keyring`, provided also by the `@polkadot/api` library, to generate key pairs that are kept in memory and disappear at the end of the program execution, unless saved to some persistent storage.
 
 :::warning
-Using the default keyring in production is highly discouraged as all the keys are kept in the memory and easily retrievable by malicious actors.
+Using the default keyring in production is highly discouraged as all the keys are kept in the memory and are more vulnerable to attacks by malicious actors.
```

Comment:
I think I asked that already, but is it really discouraged to use the polkadot keyring? What is it for then?

Reply:
The thing is that the keyring by itself keeps everything in memory. Production-ready keyrings like Sporran encrypt it behind a password, at least.